RELEASING.rst

```diff
@@ -25,6 +25,17 @@ You will need to update it in:
 
 You can also use `commitizen <https://github.com/commitizen-tools/commitizen>`_ to update the version in these files
 
+Install commitizen on mac
+
+```
+brew install commitizen
+```
+
+Bump versions locally
+
+```
+make ASTRO_PROVIDER_VERSION=(1.0.0 | 1.1.0 | 1.1.1) bump-version
```

Comment:
Sorry, I'm a bit confused by `(1.0.0 | 1.1.0 | 1.1.1)`. What does that mean?

Reply:
yep, it's confusing. I'll clean up it. But I mean you can pass major or minor or patch version

Reply:
yes, we can do that. in fact, I did that first https://github.com/astronomer/astronomer-providers/pull/1014/commits/f09b613968fdf057f6b27f810d8ce63753c39f1e but let's say my dev version is 1.7.0-dev1 and if I'll run `cz bump --increment PATCH` it set version to 1.7.1 which is problematic if my last release was 1.6.5 and I want to release 1.6.6, is this correct I'm missing something?

Reply:
I have tried to update a bit, PTAL